### PR TITLE
fortran-rpn-calc: new port

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

New port: https://github.com/scivision/rpn-calc-fortran

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
```
--->  Testing fortran-rpn-calc
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_fortran_fortran-rpn-calc/fortran-rpn-calc/work/build" && /usr/bin/make test 
Running tests...
/opt/local/bin/ctest --force-new-ctest-process 
Test project /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_fortran_fortran-rpn-calc/fortran-rpn-calc/work/build
    Start 1: RPNfunc
1/3 Test #1: RPNfunc ..........................   Passed    0.01 sec
    Start 2: UI
2/3 Test #2: UI ...............................   Passed    0.10 sec
    Start 3: Value
3/3 Test #3: Value ............................   Passed    0.10 sec

100% tests passed, 0 tests failed out of 3

Total Test time (real) =   0.22 sec
```

File: fortran/fortran-rpn-calc/Portfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+github.setup        scivision rpn-calc-fortran ec22a9780034188a691c4ce31b1778e5acf909ff
+version             1.4.1
+name                fortran-rpn-calc
+revision            0
+categories          fortran math
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         RPN Reverse Polish Notation Calculator, in Fortran
+long_description    {*}${description}
+homepage            http://scivision.github.io/rpn-calc-fortran
```
Comment (herbygillot) [inline fortran/fortran-rpn-calc/Portfile:17]: All links should be `https://` where possible

Comment (barracuda156) [reply fortran/fortran-rpn-calc/Portfile:17]: Didn’t notice, sorry. Will fix now.

**Comment by herbygillot** [comment]
Thank you; merged.

**Pull request merged**